src/kindlemint/agents/ai_teammates/development_team_orchestrator.py

```
@@ -0,0 +1,866 @@
+{str(e)}
+```
+
+**Manual intervention required.** The team decision remains: **{team_decision.final_decision.upper()}**
+
+Please enable merge manually or contact repository administrators.
+"""
+
+            try:
+                pr.create_issue_comment(failure_comment)
+            except:
+                pass  # Don't fail on comment failure
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)